commonlibktx/src/androidTest/java/com/crisolutions/commonlibraryktx/TestExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package com.crisolutions.commonlibraryktx
+
+import org.junit.Assert
+
+fun fail() {
```

Comment:
This should already exist - http://junit.sourceforge.net/javadoc/org/junit/Assert.html#fail()

Reply:
I added an equivalent `pass` function in the latest commits, should I still entirely remove the fail ?

Reply:
I think it would be best to use the already existing fail

Reply:
Removed and using fail from junit.